CMakeLists.txt

```diff
@@ -0,0 +1,459 @@
+#=============================================================================
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+cmake_minimum_required(VERSION 3.22.1 FATAL_ERROR)
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.13)
+  cmake_policy(SET CMP0077 NEW)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+endif()
+
+##############################################################################
+# - Download and initialize RAPIDS CMake helpers -----------------------------
+
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.08/RAPIDS.cmake
```

Comment:
I had some issues with this when I was working offline. The file was already downloaded and my build system was already configured. CMake tried to download again, but I had no internet connection. This clobbered the existing file, leading to an empty RAPIDS.cmake and therefore foobarred everything.  If you configure, turn off wifi/disconnect, and re-configure can you reproduce?

I don't see an option to the `file(...` command to skip if the file already exists.